packages/cli/src/utils/convert.js

```diff
@@ -213,7 +213,7 @@ const renderAuthData = (appDefinition) => {
   const fieldKeys = getAuthFieldKeys(appDefinition);
   const lines = _.map(fieldKeys, (key) => {
     const upperKey = _.snakeCase(key).toUpperCase();
-    return `${key}: process.env.${upperKey}`;
+    return `"${key}": process.env["${upperKey}"]`;
```

Comment:
`upperKey` is snake-cased. Is the double quoting still necessary? But I guess it won't hurt either. 🤷‍♂️

Reply:
oh, right you are. I'll pull it out. keeps it a little cleaner.